scripts/build/build.sh

```diff
@@ -43,13 +43,3 @@ DOTNET_HOME=$STAGE2_DIR DOTNET_TOOLS=$STAGE2_DIR $REPOROOT/scripts/test/runtests
 
 header "Validating Dependencies"
 $REPOROOT/scripts/test/validate-dependencies.sh
```

Comment:
This approach is much cleaner :) However, if we are separating package from build then please replicate the same in the PS1 in this checkin. Let's not let them drift.